frame/elections-phragmen/src/lib.rs

```diff
@@ -1775,6 +1790,49 @@ mod tests {
 		});
 	}
 
+	#[test]
+	fn voting_reserves_bond_per_vote_for_computational_effort() {
+		ExtBuilder::default().balance_factor(1000000000).voter_bond_weight_factor(1).build_and_execute(|| {
```

Comment:
these tests will break every time we re-benchmark the pallet. Better is to make the `type WeightInfo` mock-able. You can use this as an inspiration: